components/event-receiver/event-input-adapters/org.wso2.carbon.event.input.adapter.websocket.local/src/main/java/org/wso2/carbon/event/input/adapter/websocket/local/WebsocketLocalEventAdapterFactory.java

```diff
@@ -45,6 +45,11 @@ public String getType() {
     }
 
     @Override
+    public String getUsageTips() {
+        return WebsocketLocalEventAdapterConstants.ADAPTOR_USAGE_TIPS_WEBSOCKET_LOCAL;
```

Comment:
@thiliA 
Why have we hard coded the usage tip for websocket local while it is taken from the resource file for other adapters? If there is no technical limitation, let's keep it consistent across all adapters. Keeping the message in the resource file is the correct approach.